README.md

```diff
@@ -68,7 +68,16 @@ contextOptions.Permissions = new[] { ContextPermission.Geolocation };
 var context = await browser.NewContextAsync(contextOptions);
 var page = await context.NewPageAsync();
 await page.GoToAsync("https://maps.google.com");
-page.ClickAsync("text='Your location'"); //
+await page.ClickAsync("text='Your location'");
+await page.WaitForLoadStateAsync(LifecycleEvent.Networkidle);
+
+try {
+    await page.ClickAsync("text=STAY ON WEB");
```

Comment:
We could check if `QuerySelectorAsync("text=STAY ON WEB") != null` instead of using a try catch

Reply:
Done. removed try-catch block. Noticed that modal can alternatively have button with test NOT RIGHT NOW 😄.

Reply:
Do you think it is worth to commit these examples in the repository?

Reply:
If you can find a more stable example of geolocalization that would be awesome. Those texts in that page are localized, so it will work only in English.
Do you think you can replace it with a better example?

Reply:
I will work on this over the weekend and see if I can come up with something.

Reply:
I've used the class query selector instead of text.